modules/@apostrophecms/util/index.js

```diff
@@ -29,7 +29,14 @@ const Promise = require('bluebird');
 const util = require('util');
 
 module.exports = {
-  options: { alias: 'util' },
+  options: {
+    alias: 'util',
+    // No more than 50 combined levels deep in:
+    // * relationship fetches
+    // * widget loaders
+    // * async components
+    stackLimit: 50
```

Comment:
I did not try it but I guess one could override this at the project level. Is this correct?

Reply:
Yes, you can.